cmd/gateclient/client.go

```diff
@@ -466,9 +468,17 @@ func (m *GatewayClient) authenticateLdap() error {
 	auth := m.Config.Auth
 	if auth != nil && auth.Enabled && auth.Ldap != nil {
 		if !auth.Ldap.IsValid() {
-			return errors.New("Incorrect LDAP auth configuration. Must include username and password.")
+			return errors.New("Incorrect LDAP auth configuration. Must include username.")
```

Comment:
I moved the validation below the interactive prompts so that it will check for empty string being entered for username or password. I also reversed the noOp on the validator. Good catch 👍